wordpress-mobile/gutenberg-mobile
# Fix image upload progress not being displayed on the second time

To fix: wordpress-mobile/gutenberg-mobile#809

gutenberg PR: https://github.com/WordPress/gutenberg/pull/14799

After the fix:
![image-block-progress-fixed](https://user-images.githubusercontent.com/5032900/55496785-7c9feb00-5648-11e9-952e-59c370340f63.gif)

To Test:

Test with both **WPiOS** and **WPAndroid**

Test 1 - the main issue

- Add an image block
- Upload an image to it
- See the upload progress, wait until it is finished
- Tap the pencil button in the toolbar
- Select a new image to upload
- Verify that upload progress shown again just like the first try

Test 2 - multiple image blocks

- Add 2 image blocks
- Upload an image to one of them
- See the upload progress, wait until it is finished
- Verify that in the end other image block is not affected

Test 3 - parallel uploads

- Add 2 image blocks
- Upload images to both
- See the upload progress, wait until it is finished on both blocks
- Verify that uploads are finishing without problem
